pkgs/by-name/ap/appflowy/package.nix

```
@@ -17,11 +17,11 @@ let
     rec {
       x86_64-linux = {
         urlSuffix = "linux-x86_64.tar.gz";
-        hash = "sha256-fn1UK8+7+vFL4nTnnRbfjCgttVYSw6pmmqabeHqTY3g=";
+        hash = "sha256-+8uHp4YkW/G2C618QPcd28zidvY3cLJuhZoSyIHRlDE=";
```

Update to latest version
```suggestion
        hash = "sha256-1AaFl49ia/hrPNEP12AFZdugcf4IQdjs/JTRGlUOD20=";
```